dotCMS/src/main/java/com/dotmarketing/util/ConvertToPOJOUtil.java

```diff
@@ -0,0 +1,364 @@
+package com.dotmarketing.util;
+
+import com.dotcms.repackage.org.apache.commons.beanutils.PropertyUtils;
+import com.dotmarketing.beans.Identifier;
+import com.dotmarketing.portlets.containers.model.Container;
+import com.dotmarketing.portlets.folders.model.Folder;
+import com.dotmarketing.portlets.links.model.Link;
+import com.dotmarketing.portlets.templates.model.Template;
+import com.google.common.base.CaseFormat;
+import java.lang.reflect.Constructor;
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+/**
+ * Utility class used to map query results to POJO objects
+ * @author nollymar
+ */
+public class ConvertToPOJOUtil {
+
+    public static final DateFormat df = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
+
+    /**
+     * Creates new instances of T given a List<Map> of DB results
+     * @param results
+     * @param classToUse
+     * @param <T>
+     * @return
+     * @throws Exception
+     */
+    public static<T> List<T> convertDotConnectMapToPOJO(List<Map<String,String>> results, Class classToUse)
+            throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)